### PR TITLE
Set up a back up plan for showing the live feed when the internet disconnects

### PR DESCRIPTION
**Pull request by fungjj92**
## Overview

The museum wifi is supposedly spotty. The app operates offline, except for the live video feed feature which streams jpeg images from the fishway camera. We'll want to handle if the internet cuts out during a user session. 

1. User enters the See The Fishway tab for the first time and is offline --> don't show the live feed feature at all. Rationale: Don't tempt a user with features they can't access when not necessary.
  --> If the app connects to the internet while in this view, show the live feed as an option: a little abrupt and confusing perhaps, but gives the user full features avail.
2. User is on the See The Fishway tab when the internet disconnects --> present the live feed option still, but show an error message
3. The User has seen the error and revisits the tab --> always show the live feed option or the error because cat's out of the bag on it and the use should get consistent messaging

Connects #77 

### Demo

These gifs don't capture the app at its correct dimensions, so ignore styling.

Case 1
offline_to_online

Case 2
online_to_offline

Case 3
error_msg_persists

### Notes

I thought that the `onError` event listener avail to the `img` tag to detect a dropped photo stream would be the winning path forward, but that event listener never fired when the wifi was cut.

The error message is pretty placeholder. Talked through this with @alexelash and believe she'll style it later.

Sometimes, particularly when the live feed comes back online, the image is frozen for quite some time. Sometimes it comes back speedily. I don't know why this is exactly, but it is not ideal :/

## Testing Instructions

Test locally because of CORS issues with the ip streaming on Netlify.
To test the 3 cases, and any other you may think of, turn off/on your computer wifi.

### CODE REVIEW

File: src/app/src/components/VideoPlayer.js [hunk 1]
```diff
@@ -94,7 +94,7 @@ const StyledVideo = styled(Video)`
 `;
 
 const VideoPlayer = ({ src }) => {
-    const [isPlaying, setIsPlaying] = useState(true);
+    const [isPlaying, setIsPlaying] = useState(false);
```
Comment (alexelash) [inline src/app/src/components/VideoPlayer.js]: Sorry if there was some confusion. The autoplay on initial open is good, but when the user interacts with the scrubber then the video should pause. We probably should just roll back this commit.

**Review by alexelash — CHANGES_REQUESTED** [review]
Looks good and seems to feel "good" despite these being unfortunate cases! I added a comment about the autoplay. Happy to style in this branch or incorporate into a new one—let me know!

File: src/app/src/components/VideoPlayer.js (hunk 1 above)
Comment (fungjj92) [reply src/app/src/components/VideoPlayer.js]: OK thanks for clarifying! What a mess I made. The real problem turned out to be that the video controls would go out of sync to the video, see commit msg b4bafb6, which I've fixed. I had accidentally broken autoplay in this PR, which you've caught, which is also fixed in b4bafb6

**Review by caseycesari — APPROVED** [review]
All scenarios are working well.

Just one comment. Consider hiding this caption when the live feed is unavailable:

Screen Shot 2019-07-22 at 12 56 46 PM

**Comment by fungjj92** [comment]
Thanks for review and good suggestion! Here is the change as a screenshot:

Screen Shot 2019-07-24 at 9 31 53 AM

I am officially commenting out the feature now that it is complete.

**Pull request merged**